csrc/activation_kernels.cu

```diff
@@ -1,6 +1,7 @@
 #include <torch/extension.h>
 #include <ATen/cuda/CUDAContext.h>
 
+#include "cuda_compat.h"
```

Comment:
That file doesn't exist.

Reply:
Sorry, added! Is there anything else missing that prevents you from building this? If yes please let me know :)